src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEntryPointSymbol.cs

```diff
@@ -429,6 +429,8 @@ internal override BoundBlock CreateBody(DiagnosticBag diagnostics)
             }
         }
 
+        internal sealed override bool IsNullableAnalysisEnabled() => false;
```

Comment:
Can this throw instead?

Reply:
We could but I think it's reasonable to allow calling this method.

---
In reply to: [551489097](https://github.com/dotnet/roslyn/pull/49953#discussion_r551489097) [](ancestors = 551489097)